_sass/minima.scss

```diff
@@ -1,3 +1,5 @@
+// This file is unmodified but must be included so our custom syntax-highlighting partial is read
```

Comment:
Shoot, I didn't realize I opened this file PR against my `master` branch.  I meant it to be my `markdownify` branch.  That change should not even be here.